lib/autoproj/package_manifest.rb

```diff
@@ -25,11 +26,16 @@ def self.load(package, file)
         # @param [PackageDescription] the package we're loading it for
         # @param [String] contents the manifest.xml contents as a string
         # @param [String] path the file path, used for error reporting
+        # @param [Boolean] ros_manifest whether the file follows the ROS format
         # @return [PackageManifest]
         # @see load
-        def self.parse(package, contents, path: '<loaded from string>')
+        def self.parse(package, contents, path: '<loaded from string>', ros_manifest: false)
```

Comment:
I would prefer that you pass the manifest loader, either directly as a class, or as a factory function (the latter would be probably better since it does not force the loader to have the same `#initialize` signature).

Reply:
No problem, I can do that...